tests/testpkgs/savanna.nix

```diff
@@ -9,7 +13,7 @@ stdenv.mkDerivation rec {
     sha256 = "0ajc9h023vzpnlqqjli4wbvs0q36nr5p9msc3wzbic8rk687qcxc";
   };
 
-  buildInputs = [ gamin ];
+  buildInputs = [gamin];
```

Comment:
Why drop the spaces here? It makes lists more cramped and everything feels much tighter which makes the code harder to read.